docs/submission/submitting-molecular-data2.md

```diff
@@ -0,0 +1,439 @@
+:::
+
+The metadata payload is broken down into 5 sections: `root`, `experiment`, `samples`, `read groups` and `files`. Each section **must** be submitted in the payload.
+
+
+| Payload Field    | Attribute                                               | Description                                                                                                                                                                 | Permissible Values    |
+| ---------------- | ------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------- |
+| studyId          | ![Required](/assets/submission/dictionary-required.svg) | ARGO `Program ID`, the unique identifier of your program. If you have logged into the ARGO Data Platform, this is the Program ID that you see in the Program Services area. |                       |
+| analysisType     | ![Required](/assets/submission/dictionary-required.svg) | The type of molecular data that is being submitted in the payload.                                                                                                          | sequencing_experiment |
+| submitter_sequencing\_ experiment_id | ![Required](/assets/submission/dictionary-required.svg) | The unique identifier of the sequencing experiment.                                                    |                                  |
+| platform                             | ![Required](/assets/submission/dictionary-required.svg) | The sequencing platform type used in data generation.                                                  |CAPILLARY, LS454, ILLUMINA, SOLID, HELICOS, IONTORRENT, ONT, PACBIO, Nanopore, BGI|
+| platform_model                       |                                                         | The model number of the sequencing machine used in data generation.                                    |                                  |
+| experimental_strategy                | ![Required](/assets/submission/dictionary-required.svg) | The primary experimental method. For sequencing data it refers to how the sequencing library was made. | WGS, WXS, RNA-Seq, Bisulfite-Seq |
+| sequencing_date                      |                                                         | Date sequencing was performed.     
+| read_group_count | ![Required](/assets/submission/dictionary-required.svg) | The number of read groups in the molecular files being submitted submitted.                                                                                                              | |
+|read_group_id_in_bam| | Optional field indicateing the readgroup id that is in the BAM.  This cannot be submitted for FASTQ files.|                                                                    |
```

Comment:
spelling indicateing - should be indicating

Should readgroup be read group?

Reply:
done